test/functional/banner_setting_controller_test.rb

```diff
@@ -9,103 +9,102 @@ def setup
     @response   = ActionController::TestResponse.new
     User.current = nil
     @request.session[:user_id] = 1 # admin
-    
+
     # plugin setting
     @plugin = Redmine::Plugin.find('redmine_banner')
     @partial = @plugin.settings[:partial]
-    @settings = Setting["plugin_#{@plugin.id}"]    
+    @settings = Setting["plugin_#{@plugin.id}"]
   end
-  
+
   def test_get_banner_settings
-    get :plugin, :id => "redmine_banner"   
-    assert_response :success 
+    get :plugin, id: 'redmine_banner'
+    assert_response :success
     assert_template 'settings/plugin'
-    assert_select 'h2', /Redmine Banner plugin/, "#{@response.body}"
+    assert_select 'h2', /Redmine Banner plugin/, @response.body.to_s
   end
 
   def test_get_banner_settings_with_bad_format
     # set bad format
-    @settings['start_ymd'] = "1999-13-40"
-    @settings['start_ymd'] = "2011-1-35"
-    @settings['use_timer'] = "true"
-    
-    get :plugin, :id => "redmine_banner"   
-    assert_response :success 
+    @settings['start_ymd'] = '1999-13-40'
+    @settings['start_ymd'] = '2011-1-35'
+    @settings['use_timer'] = 'true'
+
+    get :plugin, id: 'redmine_banner'
+    assert_response :success
     assert_template 'settings/plugin'
     assert_select 'input[type=checkbox][id=settings_use_timer][value=true]'
-    assert_select 'h2', /Redmine Banner plugin/, "#{@response.body}"
+    assert_select 'h2', /Redmine Banner plugin/, @response.body.to_s
   end
-  
+
   def test_get_banner_settings_with_good_format
     # set bad format
-    @settings['start_ymd'] = "1999-12-31"
-    @settings['end_ymd'] = "2001-01-05"
-    @settings['use_timer'] = "true"
-    
-    get :plugin, :id => "redmine_banner"   
-    assert_response :success 
+    @settings['start_ymd'] = '1999-12-31'
+    @settings['end_ymd'] = '2001-01-05'
+    @settings['use_timer'] = 'true'
+
+    get :plugin, id: 'redmine_banner'
+    assert_response :success
     assert_template 'settings/plugin'
     assert_select 'input[type=checkbox][id=settings_use_timer][value=true]'
     assert_select 'h2', /Redmine Banner plugin/
   end
-  
+
   def test_post_banner_settings_with_good_format
     # set good format
     now = Date.today
     afterDay = now + 1
 
-    post :plugin, :id => "redmine_banner",
-      :settings => {:end_ymd => afterDay.strftime("%Y-%m-%d"), :end_min => "03", :start_min => "00", :start_hour => "00",
-      :enable => "true", :type => "warn", :display_part => "both", 
-      :start_ymd => now.strftime("%Y-%m-%d"), :use_timer => "true",
-      :banner_description => "exp. Information about upcoming Service Interruption.", 
-      :end_hour => "23"}
+    post :plugin, id: 'redmine_banner',
+                  settings: { end_ymd: afterDay.strftime('%Y-%m-%d'), end_min: '03', start_min: '00', start_hour: '00',
```

Comment:
__[rubocop]__
* Line is too long. [119/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)